app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -149,6 +154,7 @@ class Core(private val context: Context) {
     }
 
     companion object {
+        private const val BOOKMARKS_FILENAME = "bookmarks.db"
```

Comment:
Bookmarks are in the same db file as history; this should re-use connection management stuff that history is built on. See my a-c comments.